### PR TITLE
135 api integration

### PR DESCRIPTION
**Pull request by tanmaythole**
API Integration for organization routes

Ticket Link:
https://trello.com/c/AxSSnUQH/135-api-integration-of-multitenancy

### CODE REVIEW

File: src/PrivateRoutes.jsx [hunk 1]
```diff
@@ -1,30 +1,55 @@
...
-  const { isLoggedIn } = useSelector(mapState);
-  let dispatch = useDispatch();
+  const dispatch = useDispatch();
+  const location = useLocation();
+
+  const { isLoggedIn, organization, isPersonalAccount } = useSelector(mapState);
 
   useEffect(() => {
     isLoggedIn && dispatch(getUserProfileRequest());
   }, []);
```
Comment (tanmaythole) [inline src/PrivateRoutes.jsx]: add `isLoggedIn` in dependency array

Comment (tanmaythole) [reply src/PrivateRoutes.jsx]: 👍🏻

File: src/Components/OrganizationsComponents/OrgProfile/OrgProfileForm.jsx [hunk 2]
```diff
@@ -10,10 +10,10 @@ import { isValidEmail } from "utils/helper";
 import { DESCRIPTION, EMAIL, ORG_NAME } from "constants/userMessagesConstants";
 
 const OrgProfileForm = ({ data, onChange, onSubmit, isOrgOwner }) => {
-  const { name, description, email } = data;
+  const { name, description, contactEmail } = data;
```
Comment (ChinmaySonawane) [inline src/Components/OrganizationsComponents/OrgProfile/OrgProfileForm.jsx:13]: contactEmail and email are different?

File: src/containers/DashBoard/index.jsx [hunk 3]
```diff
@@ -13,44 +13,48 @@ import { addProjectRequest, getProjectsRequest } from "store/Projects/actions";
...
+  // toggles the modal
   const toggleModal = () => {
     setShowAddProjectModal(!showAddProjectModal);
   };
 
+  // handles when project form data changes
   const onChangeProjectFormData = (e) => {
     setAddProjectFormData((prevState) => ({
       ...prevState,
       [e.target.name]: e.target.value,
```
Comment (ChinmaySonawane) [inline src/containers/DashBoard/index.jsx:46]: why we are storing the data in key value format?

File: src/Components/OrganizationsComponents/OrgProfile/OrgProfileForm.jsx (hunk 2 above)
Comment (tanmaythole) [reply src/Components/OrganizationsComponents/OrgProfile/OrgProfileForm.jsx:13]: The variable names were just updated so as not to get confused over data coming from the backend and stored in state.

File: src/containers/DashBoard/index.jsx (hunk 3 above)
Comment (tanmaythole) [reply src/containers/DashBoard/index.jsx:46]: Because we manage form data into the state instead of reducer

**Review by ChinmaySonawane — APPROVED** [review]

**Pull request merged**